src/i18n/en/nav.ts

```diff
@@ -18,190 +18,105 @@
-	},
-
-	{ text: 'Integrations', header: true, type: 'learn', key: 'addons' },
-	{ text: 'Add integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
-	{
-		text: 'UI Frameworks',
-		slug: 'guides/framework-components',
-		key: 'guides/framework-components',
-	},
-	{
-		text: 'SSR Adapters',
-		slug: 'guides/server-side-rendering',
-		key: 'guides/server-side-rendering',
-	},
-	{
-		text: 'Astro DB',
-		slug: 'guides/astro-db',
-		key: 'guides/astro-db',
-	},
-
-	{ text: 'Recipes', header: true, type: 'learn', key: 'examples' },
-	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
-	{ text: 'Connect to Astro Studio', slug: 'recipes/studio', key: 'recipes/studio' },
-	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
-	{ text: 'Add backend services', slug: 'guides/backend', key: 'guides/backend' },
-	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
-	{ text: 'More recipes', slug: 'recipes', key: 'guides/recipes' },
+	{ text: 'Astro Template Syntax', slug: 'basics/astro-syntax', key: 'basics/astro-syntax' },
+	{ text: 'Rendering Modes', slug: 'basics/rendering-modes', key: 'basics/rendering-modes' },
+	{ text: 'Imports', slug: 'guides/imports', key: 'guides/imports' },
+	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
 
-	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
+	
+	{ text: 'Create Routes', header: true, type: 'learn', key: 'routes' },
 	{ text: 'Routing', slug: 'guides/routing', key: 'guides/routing' },
+	{ text: 'Endpoints', slug: 'guides/endpoints', key: 'guides/endpoints' },
+	{ text: 'Prefetch', slug: 'guides/prefetch', key: 'guides/prefetch' },
+	{ text: 'Middleware', slug: 'guides/middleware', key: 'guides/middleware' },
+	{ text: 'Internationalization', slug: 'guides/internationalization', key: 'guides/internationalization' },
+	{ text: 'View Transitions', slug: 'guides/view-transitions', key: 'guides/view-transitions' },
+
+
+	{ text: 'Manage your content', header: true, type: 'learn', key: 'content' },
+	{ text: 'Authoring Content', slug: 'guides/content', key: 'guides/content' },
 	{ text: 'Markdown', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
-	{
-		text: 'Scripts & Event Handling',
-		slug: 'guides/client-side-scripts',
-		key: 'guides/client-side-scripts',
-	},
+	{ text: 'Content Collections', slug: 'guides/content-collections', key: 'guides/content-collections' },
+	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Add an RSS feed', slug: 'guides/rss', key: 'guides/rss' },
+
+
+
+	{ text: 'Connect your data', header: true, type: 'learn', key: 'data' },
+	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
+	{ text: 'Astro DB', slug: 'guides/astro-db', key: 'guides/astro-db' },
+	{ text: 'Add backend services', slug: 'guides/backend', key: 'guides/backend' },
```

Comment:
```suggestion
	{ text: 'Add Backend Services', slug: 'guides/backend', key: 'guides/backend' },
```

Reply:
In fact, this matches more with "Connect your data" and it might be "Data Fetching" that's out of place. We'll take one sweep for page titles!